### DISCUSSION TITLE
Error during execution of "Show PK-Analysis" in population simulation

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1593] [Questions & Problems] [answered]
Hello, everyone.
I'm getting an error when executing "Show PK-Analysis" in a population simulation. This error seems to have occurred before, ref: https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2632
I found a solution in the forum, but it's too much trouble for a lot of data, please fix this error again, thanks!

**Comment by Yuri05** [comment, marked as answer]
This issue was fixed, and the fix will be available in the upcoming version 12 of the OSP Suite.
In the meanwhile, you can try the workaround described [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2632#issuecomment-1560694547)

**Reply by SCZ1174** [reply]
That's great!
I've solved my problem using the method in the link, thanks!
